lib/report_formatter/timeline.rb

```diff
@@ -175,25 +175,6 @@ def tl_event(tl_xml, row, col)
       # manipulating column order to display timestamp at the end of the bubble.
       field = mri.timeline[:field].split("-")
       if ems && ems_cloud
-        # Remove Infra specific fields
```

Comment:
@h-kataria These lines where made redundant by this patch.
These values are handeld below e.g "host_name" in L218 we will reach the else block in 258 and in the end in 269 nothing will be written.